### PR TITLE
Updates UI on Category Listing page.

### PR DESCRIPTION
**Pull request by Shrikant9**
Issue [#357](https://github.com/bigbinary/acehelp/issues/357) Makes categories listing UI to incorporate the new design.

### CODE REVIEW

**Comment by Shrikant9** [comment]
@breezyboa _a

File: app/javascript/Admin/Page/Category/List.elm [hunk 1]
```diff
@@ -106,8 +108,8 @@ view model =
                 ]
                 [ text "New Category" ]
             ]
-        , div
-            []
+        , table
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Category/List.elm]: Can we avoid using tables

**Comment by Shrikant9** [comment]
@rishimohan _a

**Comment by rishimohan** [comment]
Looks good to me, @breezyboa please merge if rest of the changes are good.

**Pull request merged**